lusidtools/general_utils/quick_func.py

```diff
@@ -0,0 +1,302 @@
+# This file contains a set of data manipulation functions
+# These functions are primarily used in the sample-notebooks project
+# See the following URL for further details: https://github.com/finbourne/sample-notebooks
+
+import time as default_time
+import pandas as pd
+import numpy as np
+import pytz
+from datetime import datetime, timedelta, time
+from random import randint
+
+def create_scope_id():
+    """
+    This function creates a unique ID based on the time since epoch for use
+    as a scope id.
+
+    Output
+    scopes: Scope identifier
+    """
+    # Get the current time since epoch
+    test = default_time.time()
+    # Multiply by 7 to get value to 100s of nano seconds
+    timestamp = hex(int(test *10000000.0))
+    # Create the scope id by joining the hex representation with dashes every 4 charachters
+    scope_id = '-'.join(timestamp[i: i +4] for i in range(2, len(timestamp), 4))
+    return scope_id
+
+
+def import_file(csv_file):
+    """
+    This function is used to import data form our files
+    """
+
+    data = pd.read_csv(csv_file)
+
+    return data
+
+
+def fetch_portfolio_names(csv_file):
+    """
+    This function loads the data from our portfolio names file and writes it to a dictionary
+
+    Input
+    csv_file: The name of the csv file including the extension i.e. .csv
+
+    Output
+    client_portfolios: Dictionary containing the client portfolios
+    """
+
+    portfolios = import_file(csv_file)
+
+    client_portfolios = {}
+
+    for index, portfolio_group, portfolio in portfolios.itertuples():
+        client_portfolios.setdefault(portfolio_group, []).append(portfolio)
+
+    return client_portfolios
+
+
+def fetch_portfolio_names(csv_file):
+    """
+    This function loads the data from our portfolio names file and writes it to a dictionary
+
+    Input
+    csv_file: The name of the csv file including the extension i.e. .csv
+
+    Output
+    client_portfolios: Dictionary containing the client portfolios
+    """
+
+    portfolios = import_file(csv_file)
+
+    client_portfolios = {}
+
+    for index, portfolio_group, portfolio in portfolios.itertuples():
+        client_portfolios.setdefault(portfolio_group, []).append(portfolio)
+
+    return client_portfolios
+
+
+def fetch_instrument_universe(csv_file, paper=False):
```

Comment:
`paper=False` is a bit specific and horrible, is it possible parameterise the function instead?